File: mocha/prime.js
let num=prompt("Enter a number");
function is_prime(num){
    if (num<=1){
        alert("its not prime number")
        return;
    }
    else{
       let i=2;
        while(i<num){
            if(num%2==0){
                alert("its not prime number")
                return;
            }
            i++;
            
        }
        

            alert("its  prime number")
        
    }
}
is_prime(num);

File: python_q/prime.py
num=10
l=[]
while num<100:
   
    if num<2:
     pass
    else:
        i=2
        while i<num:
            if num%i==0:
                
                break
            i+=1
        else:
            l.append(num)
    num+=1
print(l)
